beego.go

```diff
@@ -73,13 +72,6 @@ func initBeforeHTTPRun() {
 	if err != nil {
 		panic(err)
 	}
-	//init log
-	for adaptor, config := range BConfig.Log.Outputs {
-		err = BeeLogger.SetLogger(adaptor, config)
-		if err != nil {
-			fmt.Printf("%s with the config `%s` got err:%s\n", adaptor, config, err)
-		}
-	}
```

Comment:
why I add these init log here. As user can change the config path. So if they change that, we need to init the BeeLogger again.